docs/source/operations/projections/peirce_q.rst

```diff
@@ -21,12 +38,33 @@ Peirce Quincuncial
 +---------------------+----------------------------------------------------------+
 
 
-.. figure:: ./images/peirce_q.png
+.. figure:: ./images/peirce_q_square.png
+   :width: 500 px
+   :align: center
+   :alt:   Peirce Quincuncial (Square)
+
+   proj-string: ``+proj=peirce_q +lon_0=25 +type=square``
+
+.. figure:: ./images/peirce_q_diamond.png
+   :width: 500 px
+   :align: center
+   :alt:   Peirce Quincuncial (Diamond)
+
+   proj-string: ``+proj=peirce_q +lon_0=25 +type=diamond``
+
+.. figure:: ./images/peirce_q_horizontal.png
+   :width: 500 px
+   :align: center
+   :alt:   Peirce Quincuncial (Horizontal)
+
+   proj-string: ``+proj=peirce_q +lon_0=25 +type=horizontal``
+
+.. figure:: ./images/grieger_triptychial.png
    :width: 500 px
    :align: center
-   :alt:   Peirce Quincuncial
+   :alt:   Grieger Triptychial
 
-   proj-string: ``+proj=peirce_q``
+   proj-string: ``+proj=pipeline +step +proj=ob_tran +o_proj=peirce_q +o_lat_p=-45 +o_lon_p=45 +o_type=horizontal +o_scrollx=-0.25 +step +proj=affine +s11=-1 +s12=0 +s21=0 +s22=-1``
```

Comment:
so we do need to remove the o_ prefixes here:
```suggestion
   proj-string: ``+proj=pipeline +step +proj=ob_tran +o_proj=peirce_q +o_lat_p=-45 +o_lon_p=45 +type=horizontal +scrollx=-0.25 +step +proj=affine +s11=-1 +s12=0 +s21=0 +s22=-1``
```

Reply:
I think this should remain as is: o_type and o_scrollx do not get passed by ob_tran to peirce_q, but plain type and scrollx do . Otherwise fix needed in ob_tran?

Reply:
> I think this should remain as is: o_type and o_scrollx do not get passed by ob_tran to peirce_q, but plain type and scrollx do . Otherwise fix needed in ob_tran?

I believe we agree, but your sentence contradicts my suggested fix. My suggested fix consists in removing the o_ prefix of o_type and o_scrollx.

Reply:
yep, i we agree. i think all is as it should be here

Reply:
The original proj-string `+proj=peirce_q` or more specifically `+proj=peirce_q +type=crs` now returns an error, while it didn't do this before this patch.